resources/views/pages/loto.blade.php

```diff
@@ -1,26 +1,32 @@
 @extends('layout')
 @section('content')
 
-    <div class="container mt-3">
-        <button class="btn btn-danger" onclick="getRandomNumbers()">Generate random combination</button>
+    <div class="container mt-5 text-center">
+        <button class="btn btn-outline-secondary" onclick="getRandomNumbers()">Generate random combination</button>
     </div>
 
-    <form id="rndNumbers" class="container mb-5 mt-3" action="{{route("loto.buy")}}" method="post">
```

Comment:
{{ csrf_field() }} line creates a hidden input, and it was being changed by the javascript because it's an input element that is inside the #rndNumbers form, I solved it by moving the rndNumbers id into the random number container.